operator-common/src/test/java/io/strimzi/operator/common/auth/PemAuthIdentityTest.java

```diff
@@ -52,13 +53,13 @@ public void testSecretCorrupted() {
                     .withName(KafkaResources.secretName(CLUSTER))
                     .withNamespace(NAMESPACE)
                 .endMetadata()
-                .withData(map("cluster-operator.key", "notacert",
-                        "cluster-operator.crt", "notacert",
-                        "cluster-operator.p12", "notatruststore",
-                        "cluster-operator.password", "notapassword"))
+                .withData(map("cluster-operator.key", MockCertManager.clusterCaKey(),
+                        "cluster-operator.crt", "bm90YWNlcnQ=",
+                        "cluster-operator.p12", "bm90YXRydXN0c3RvcmU=",
+                        "cluster-operator.password", "bm90YXBhc3N3b3Jk"))
```

Comment:
Maybe you can add comments what the base64 encoded values stand for to make it more easily readable without decoding it 😉

Reply:
done 😄